client/my-sites/backup/clone-flow/test/index.jsx

```diff
@@ -0,0 +1,468 @@
+import BackupSuccessful from 'calypso/components/jetpack/daily-backup-status/status-card/backup-successful';
+import useRewindableActivityLogQuery from 'calypso/data/activity-log/use-rewindable-activity-log-query';
+import getInProgressRewindStatus from 'calypso/state/selectors/get-in-progress-rewind-status';
+import getRewindState from 'calypso/state/selectors/get-rewind-state';
+import Error from '../../rewind-flow/error';
+import Loading from '../../rewind-flow/loading';
+import ProgressBar from '../../rewind-flow/progress-bar';
+import RewindConfigEditor from '../../rewind-flow/rewind-config-editor';
+import BackupCloneFlow from '../index';
+import CloneFlowStepProgress from '../step-progress';
+
+jest.mock( 'react', () => ( {
+	...jest.requireActual( 'react' ),
+	useState: jest.fn(),
+} ) );
+
+jest.mock( 'i18n-calypso', () => ( {
+	...jest.requireActual( 'i18n-calypso' ),
+	localize: ( x ) => x,
+	translate: ( x ) => x,
+	useTranslate: jest.fn( () => ( text ) => text ),
+} ) );
+
+jest.mock( 'calypso/data/activity-log/use-rewindable-activity-log-query', () =>
+	jest.fn().mockImplementation( () => {
+		return { isLoading: true, data: null };
+	} )
+);
+
+jest.mock( 'calypso/data/activity-log/use-rewindable-activity-log-query' );
+jest.mock( 'calypso/state/selectors/get-in-progress-rewind-status' );
+jest.mock( 'calypso/state/selectors/get-rewind-state' );
+
+jest.mock( 'calypso/state/selectors/get-site-gmt-offset', () =>
+	jest.fn().mockImplementation( () => null )
+);
+
+jest.mock( 'calypso/state/selectors/get-site-timezone-value', () =>
+	jest.fn().mockImplementation( () => null )
+);
+
+jest.mock( 'calypso/components/activity-card-list', () =>
+	jest.fn().mockImplementation( () => null )
+);
+
+jest.mock( 'calypso/components/advanced-credentials', () =>
+	jest.fn().mockImplementation( () => null )
+);
+jest.mock( 'calypso/components/jetpack/daily-backup-status/status-card/backup-successful', () =>
+	jest.fn().mockImplementation( () => null )
+);
+
+jest.mock( '../step-progress', () => jest.fn().mockImplementation( () => null ) );
+jest.mock( '../../rewind-flow/error', () => jest.fn().mockImplementation( () => null ) );
+jest.mock( '../../rewind-flow/loading', () => jest.fn().mockImplementation( () => null ) );
+jest.mock( '../../rewind-flow/progress-bar', () => jest.fn().mockImplementation( () => null ) );
+jest.mock( '../../rewind-flow/rewind-config-editor', () =>
+	jest.fn().mockImplementation( () => null )
+);
+
+function createState( siteId = 1 ) {
+	return {
+		currentUser: {
+			capabilities: {
+				[ siteId ]: {
+					publish_posts: true,
+				},
+			},
+		},
+		sites: {
+			plans: {
+				[ siteId ]: {},
+			},
+		},
+		ui: {
+			selectedSiteId: siteId,
+			section: {
+				name: 'backup"',
+				paths: [ '/backup' ],
+				module: 'calypso/my-sites/backup',
+				group: 'sites',
+			},
+		},
+		preferences: {
+			remoteValues: {},
+		},
+		documentHead: { unreadCount: 1 },
+	};
+}
+
+function getLogs() {
+	return {
```

Comment:
Maybe we could consider moving the fixtures into a separate files like [this example](https://github.com/Automattic/wp-calypso/blob/20fc989ed71ca1b6202043ed0c2a71ace965ed8e/client/reader/discover/test/fixtures/index.js) to make this file smaller.